test/contrib/einstein/test_stein_loss.py

```diff
@@ -80,7 +80,7 @@ def stein_loss_fn(chosen_particle, obs, particles, assign):
     xs = jnp.array([-1, 0.5, 3.0])
     num_particles = xs.shape[0]
     particles = {"x": xs}
-    zs = jnp.array([-0.1241799, -0.65357316, -0.96147573])  # from inspect
+    zs = jnp.array([-3.3022664, -1.06049, 0.64527285])  # from inspect
```

Comment:
could we just replicate the logic to generate zs here? @OlaRonning

Reply:
Yeah, that should be possible. I'll have a look at it now.